vault/service/clean.sls

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls syntax=yaml softtabstop=2 tabstop=2 shiftwidth=2 expandtab autoindent
+
+vault-service-clean-service-dead:
+  service.dead:
+    - name: vault
+    - enable: False
+
+vault-service-clean-file-absent:
+  file.absent:
+    - name: /etc/systemd/system/vault.service
```

Comment:
If keeping this `file.absent`, shouldn't be literal -- i.e. use `{{ vault.service.path }}`.  I say _if_, since I'm not entirely convinced this needs to be here in the first place.